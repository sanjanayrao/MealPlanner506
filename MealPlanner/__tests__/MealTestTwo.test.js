import React from 'react';
import {cleanup, fireEvent} from '@testing-library/react';
import Meal from '../components/Meals';
import Adapter from "enzyme-adapter-react-16";
import { shallow, configure, render  } from "enzyme";
import Button from '../components/Button.ios';





configure({ adapter: new Adapter() });

it("Meal Renders Same as Prior Test", () => {
    const wrapper = render(<Meal />);
    expect(wrapper).toMatchSnapshot();
  });