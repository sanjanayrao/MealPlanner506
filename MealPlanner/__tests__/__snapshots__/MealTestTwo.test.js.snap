// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Meal Renders Same as Prior Test 1`] = `
initialize {
  "0": Object {
    "attribs": Object {},
    "children": Array [
      Object {
        "attribs": Object {},
        "children": Array [
          Object {
            "attribs": Object {
              "style": "margin:15px",
            },
            "children": Array [
              Object {
                "data": "My Meals",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "text",
              },
            ],
            "name": "text",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": Object {
              "attribs": Object {
                "focusable": "true",
                "style": "background-color:#553555;padding-top:7px;border-radius:100px;width:60px;height:60px;align-self:flex-end;align-items:center;margin:10px",
              },
              "children": Array [
                Object {
                  "attribs": Object {
                    "style": "color:white;font-size:36px",
                  },
                  "children": Array [
                    Object {
                      "data": "+",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                  ],
                  "name": "text",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "style": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "style": undefined,
                  },
                },
              ],
              "name": "view",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": null,
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
              "x-attribsNamespace": Object {
                "focusable": undefined,
                "style": undefined,
              },
              "x-attribsPrefix": Object {
                "focusable": undefined,
                "style": undefined,
              },
            },
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "style": undefined,
            },
            "x-attribsPrefix": Object {
              "style": undefined,
            },
          },
          Object {
            "attribs": Object {
              "focusable": "true",
              "style": "background-color:#553555;padding-top:7px;border-radius:100px;width:60px;height:60px;align-self:flex-end;align-items:center;margin:10px",
            },
            "children": Array [
              Object {
                "attribs": Object {
                  "style": "color:white;font-size:36px",
                },
                "children": Array [
                  Object {
                    "data": "+",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                ],
                "name": "text",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "style": undefined,
                },
                "x-attribsPrefix": Object {
                  "style": undefined,
                },
              },
            ],
            "name": "view",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "style": "margin:15px",
              },
              "children": Array [
                Object {
                  "data": "My Meals",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
              ],
              "name": "text",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": [Circular],
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "style": undefined,
              },
              "x-attribsPrefix": Object {
                "style": undefined,
              },
            },
            "type": "tag",
            "x-attribsNamespace": Object {
              "focusable": undefined,
              "style": undefined,
            },
            "x-attribsPrefix": Object {
              "focusable": undefined,
              "style": undefined,
            },
          },
        ],
        "name": "view",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": Object {
          "attribs": Object {},
          "children": Array [],
          "name": "text",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": Object {
            "attribs": Object {
              "style": "align-self:center;align-items:center;margin:10px;justify-content:center;align-content:center",
            },
            "children": Array [
              Object {
                "attribs": Object {
                  "data": "",
                  "initialnumtorender": "10",
                  "maxtorenderperbatch": "10",
                  "scrolleventthrottle": "50",
                  "stickyheaderindices": "",
                  "updatecellsbatchingperiod": "50",
                  "viewabilityconfigcallbackpairs": "",
                  "windowsize": "21",
                },
                "children": Array [
                  Object {
                    "attribs": Object {},
                    "children": Array [],
                    "name": "view",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {},
                    "x-attribsPrefix": Object {},
                  },
                ],
                "name": "rctscrollview",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "data": undefined,
                  "initialnumtorender": undefined,
                  "maxtorenderperbatch": undefined,
                  "scrolleventthrottle": undefined,
                  "stickyheaderindices": undefined,
                  "updatecellsbatchingperiod": undefined,
                  "viewabilityconfigcallbackpairs": undefined,
                  "windowsize": undefined,
                },
                "x-attribsPrefix": Object {
                  "data": undefined,
                  "initialnumtorender": undefined,
                  "maxtorenderperbatch": undefined,
                  "scrolleventthrottle": undefined,
                  "stickyheaderindices": undefined,
                  "updatecellsbatchingperiod": undefined,
                  "viewabilityconfigcallbackpairs": undefined,
                  "windowsize": undefined,
                },
              },
            ],
            "name": "rctsafeareaview",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": [Circular],
            "type": "tag",
            "x-attribsNamespace": Object {
              "style": undefined,
            },
            "x-attribsPrefix": Object {
              "style": undefined,
            },
          },
          "parent": [Circular],
          "prev": [Circular],
          "type": "tag",
          "x-attribsNamespace": Object {},
          "x-attribsPrefix": Object {},
        },
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {},
        "x-attribsPrefix": Object {},
      },
      Object {
        "attribs": Object {},
        "children": Array [],
        "name": "text",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": Object {
          "attribs": Object {
            "style": "align-self:center;align-items:center;margin:10px;justify-content:center;align-content:center",
          },
          "children": Array [
            Object {
              "attribs": Object {
                "data": "",
                "initialnumtorender": "10",
                "maxtorenderperbatch": "10",
                "scrolleventthrottle": "50",
                "stickyheaderindices": "",
                "updatecellsbatchingperiod": "50",
                "viewabilityconfigcallbackpairs": "",
                "windowsize": "21",
              },
              "children": Array [
                Object {
                  "attribs": Object {},
                  "children": Array [],
                  "name": "view",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                  "x-attribsNamespace": Object {},
                  "x-attribsPrefix": Object {},
                },
              ],
              "name": "rctscrollview",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "data": undefined,
                "initialnumtorender": undefined,
                "maxtorenderperbatch": undefined,
                "scrolleventthrottle": undefined,
                "stickyheaderindices": undefined,
                "updatecellsbatchingperiod": undefined,
                "viewabilityconfigcallbackpairs": undefined,
                "windowsize": undefined,
              },
              "x-attribsPrefix": Object {
                "data": undefined,
                "initialnumtorender": undefined,
                "maxtorenderperbatch": undefined,
                "scrolleventthrottle": undefined,
                "stickyheaderindices": undefined,
                "updatecellsbatchingperiod": undefined,
                "viewabilityconfigcallbackpairs": undefined,
                "windowsize": undefined,
              },
            },
          ],
          "name": "rctsafeareaview",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": null,
          "parent": [Circular],
          "prev": [Circular],
          "type": "tag",
          "x-attribsNamespace": Object {
            "style": undefined,
          },
          "x-attribsPrefix": Object {
            "style": undefined,
          },
        },
        "parent": [Circular],
        "prev": Object {
          "attribs": Object {},
          "children": Array [
            Object {
              "attribs": Object {
                "style": "margin:15px",
              },
              "children": Array [
                Object {
                  "data": "My Meals",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
              ],
              "name": "text",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": Object {
                "attribs": Object {
                  "focusable": "true",
                  "style": "background-color:#553555;padding-top:7px;border-radius:100px;width:60px;height:60px;align-self:flex-end;align-items:center;margin:10px",
                },
                "children": Array [
                  Object {
                    "attribs": Object {
                      "style": "color:white;font-size:36px",
                    },
                    "children": Array [
                      Object {
                        "data": "+",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                    ],
                    "name": "text",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "style": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "style": undefined,
                    },
                  },
                ],
                "name": "view",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": [Circular],
                "type": "tag",
                "x-attribsNamespace": Object {
                  "focusable": undefined,
                  "style": undefined,
                },
                "x-attribsPrefix": Object {
                  "focusable": undefined,
                  "style": undefined,
                },
              },
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "style": undefined,
              },
              "x-attribsPrefix": Object {
                "style": undefined,
              },
            },
            Object {
              "attribs": Object {
                "focusable": "true",
                "style": "background-color:#553555;padding-top:7px;border-radius:100px;width:60px;height:60px;align-self:flex-end;align-items:center;margin:10px",
              },
              "children": Array [
                Object {
                  "attribs": Object {
                    "style": "color:white;font-size:36px",
                  },
                  "children": Array [
                    Object {
                      "data": "+",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                  ],
                  "name": "text",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "style": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "style": undefined,
                  },
                },
              ],
              "name": "view",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": null,
              "parent": [Circular],
              "prev": Object {
                "attribs": Object {
                  "style": "margin:15px",
                },
                "children": Array [
                  Object {
                    "data": "My Meals",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                ],
                "name": "text",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": [Circular],
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "style": undefined,
                },
                "x-attribsPrefix": Object {
                  "style": undefined,
                },
              },
              "type": "tag",
              "x-attribsNamespace": Object {
                "focusable": undefined,
                "style": undefined,
              },
              "x-attribsPrefix": Object {
                "focusable": undefined,
                "style": undefined,
              },
            },
          ],
          "name": "view",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": [Circular],
          "parent": [Circular],
          "prev": null,
          "type": "tag",
          "x-attribsNamespace": Object {},
          "x-attribsPrefix": Object {},
        },
        "type": "tag",
        "x-attribsNamespace": Object {},
        "x-attribsPrefix": Object {},
      },
      Object {
        "attribs": Object {
          "style": "align-self:center;align-items:center;margin:10px;justify-content:center;align-content:center",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "data": "",
              "initialnumtorender": "10",
              "maxtorenderperbatch": "10",
              "scrolleventthrottle": "50",
              "stickyheaderindices": "",
              "updatecellsbatchingperiod": "50",
              "viewabilityconfigcallbackpairs": "",
              "windowsize": "21",
            },
            "children": Array [
              Object {
                "attribs": Object {},
                "children": Array [],
                "name": "view",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {},
                "x-attribsPrefix": Object {},
              },
            ],
            "name": "rctscrollview",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "data": undefined,
              "initialnumtorender": undefined,
              "maxtorenderperbatch": undefined,
              "scrolleventthrottle": undefined,
              "stickyheaderindices": undefined,
              "updatecellsbatchingperiod": undefined,
              "viewabilityconfigcallbackpairs": undefined,
              "windowsize": undefined,
            },
            "x-attribsPrefix": Object {
              "data": undefined,
              "initialnumtorender": undefined,
              "maxtorenderperbatch": undefined,
              "scrolleventthrottle": undefined,
              "stickyheaderindices": undefined,
              "updatecellsbatchingperiod": undefined,
              "viewabilityconfigcallbackpairs": undefined,
              "windowsize": undefined,
            },
          },
        ],
        "name": "rctsafeareaview",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": Object {
          "attribs": Object {},
          "children": Array [],
          "name": "text",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": [Circular],
          "parent": [Circular],
          "prev": Object {
            "attribs": Object {},
            "children": Array [
              Object {
                "attribs": Object {
                  "style": "margin:15px",
                },
                "children": Array [
                  Object {
                    "data": "My Meals",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                ],
                "name": "text",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": Object {
                  "attribs": Object {
                    "focusable": "true",
                    "style": "background-color:#553555;padding-top:7px;border-radius:100px;width:60px;height:60px;align-self:flex-end;align-items:center;margin:10px",
                  },
                  "children": Array [
                    Object {
                      "attribs": Object {
                        "style": "color:white;font-size:36px",
                      },
                      "children": Array [
                        Object {
                          "data": "+",
                          "next": null,
                          "parent": [Circular],
                          "prev": null,
                          "type": "text",
                        },
                      ],
                      "name": "text",
                      "namespace": "http://www.w3.org/1999/xhtml",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "tag",
                      "x-attribsNamespace": Object {
                        "style": undefined,
                      },
                      "x-attribsPrefix": Object {
                        "style": undefined,
                      },
                    },
                  ],
                  "name": "view",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": null,
                  "parent": [Circular],
                  "prev": [Circular],
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "focusable": undefined,
                    "style": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "focusable": undefined,
                    "style": undefined,
                  },
                },
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "style": undefined,
                },
                "x-attribsPrefix": Object {
                  "style": undefined,
                },
              },
              Object {
                "attribs": Object {
                  "focusable": "true",
                  "style": "background-color:#553555;padding-top:7px;border-radius:100px;width:60px;height:60px;align-self:flex-end;align-items:center;margin:10px",
                },
                "children": Array [
                  Object {
                    "attribs": Object {
                      "style": "color:white;font-size:36px",
                    },
                    "children": Array [
                      Object {
                        "data": "+",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                    ],
                    "name": "text",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "style": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "style": undefined,
                    },
                  },
                ],
                "name": "view",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": Object {
                  "attribs": Object {
                    "style": "margin:15px",
                  },
                  "children": Array [
                    Object {
                      "data": "My Meals",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                  ],
                  "name": "text",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": [Circular],
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "style": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "style": undefined,
                  },
                },
                "type": "tag",
                "x-attribsNamespace": Object {
                  "focusable": undefined,
                  "style": undefined,
                },
                "x-attribsPrefix": Object {
                  "focusable": undefined,
                  "style": undefined,
                },
              },
            ],
            "name": "view",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": [Circular],
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {},
            "x-attribsPrefix": Object {},
          },
          "type": "tag",
          "x-attribsNamespace": Object {},
          "x-attribsPrefix": Object {},
        },
        "type": "tag",
        "x-attribsNamespace": Object {
          "style": undefined,
        },
        "x-attribsPrefix": Object {
          "style": undefined,
        },
      },
    ],
    "name": "view",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {},
    "x-attribsPrefix": Object {},
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
